desktop/plugins/network/index.tsx

```diff
@@ -385,6 +394,67 @@ export function plugin(client: PluginClient<Events, Methods>) {
 
         informClientMockChange(routes.get());
       },
+      importRoutes() {
+        const options: OpenDialogOptions = {
+          properties: ['openFile'],
+          filters: [{extensions: ['json'], name: 'Flipper Route Files'}],
+          title: 'Import Routes',
+        };
+        remote.dialog.showOpenDialog(options).then((result) => {
+          const filePaths = result.filePaths;
+          if (filePaths.length > 0) {
+            fs.readFile(filePaths[0], 'utf8', (err, data) => {
+              if (err) {
+                console.error(err);
+                return;
+              }
+              const importedRoutes = JSON.parse(data);
+              importedRoutes?.forEach((importedRoute: Route) => {
+                if (importedRoute != null) {
+                  const newNextRouteId = nextRouteId.get();
+                  routes.update((draft) => {
+                    draft[newNextRouteId.toString()] = {
```

Comment:
As @mgoyanes noted, probably the preferred behavior is to reuse routes if a route with the same url + method does exist already?

Reply:
Just to confirm, if the same route is imported we could do either of these:

- use the original definition (and ignore the new import) or
- over-ride the original definition

It seems to me that we should over-ride the original definition since the group of imported routes would be needed to test a particular scenario

Reply:
although, the user could ensure this by "clearing" all the existing routes before doing the import (now I've talked myself into the first approach)

Reply:
Right now both routes would be shown and the user could pick the one they want by deleting the other - so maybe no change needed

Reply:
As a user I'd assume the second

Reply:
@mweststrate   Thanks for merging this PR.  I still intend to implement the change for over-riding any duplicate routes.  Hopefully will have time in the next few days.